README.md

```diff
@@ -12,42 +12,19 @@ These tests can all be run at a go using the "run" file or executed via a Docker
 
 ###General Configuration###
 
+Python 3.5.2 or greater is required.
+
 It's highly recommended to use [Virtualenv](https://virtualenv.pypa.io/en/latest/)
 in order to have an isolated environment.
 
-From /kinto/
+From this directory
 
-1. virtualenv .
+1. virtualenv . -p /path/to/python
 2. ./bin/pip install -r dev-requirements.txt
 
-NOTE -- as of 01/14/2016 we are relying on a specific branch of the [Kinto Python client](https://github.com/Kinto/kinto.py)
-that implements the replication features. When the pull request containing that
-code gets merged into the master branch the dev-requirements.txt file will be
-updated.
 
 #Run Tests#
```

Comment:
Yeah, I'm pretty sure they are all config tests from QA's perspective. If we add different types of tests in we can just update the documentation. Based on what we have right now, all we have that are updated and used on a regular basis are config tests